docs/community-history.md

```diff
@@ -1,11 +1,13 @@
 # Community History
 
-<iframe class="youtube-player" type="text/html" style="width: 400px;
-  height: 300px" src="https://www.youtube.com/embed/-zRN7XLCRhc"
-  frameborder="0">
-</iframe>
-
+<!-- markdownlint-disable no-inline-html -->
```

Comment:
We want these to be as narrow as possible. So the disable should be immediately before the div, and there should be a corresponding enable immediately after the div.